mreg_cli/outputmanager.py

```diff
@@ -394,14 +397,14 @@ def get_filter(self, command: str) -> tuple[str, Any, bool]:
                 try:
                     filter_re = re.compile(filter_str)
                 except re.error as exc:
-                    if "|" in command_issued:
-                        raise InputFailure(
-                            "Command parts that contain a pipe ('|') must be quoted.",
-                        ) from exc
+                    base_msg = f"Unable to compile regex '{filter_str}'"
+                    msg = build_error_message(command_issued)
+                    if msg:
+                        # Custom error messages with suggestions go on their own line
+                        raise CliError(f"{base_msg}\n{msg}") from exc
```

Comment:
This would probably at some point be factored so one doesn't have to manually remember the newline.

Reply:
Resolved in [a62ad56](https://github.com/unioslo/mreg-cli/pull/300/commits/a62ad561a684dfac634892ad17172ebf2a768c14)